stingray/modeling/parameterestimation.py

```diff
@@ -794,3 +877,76 @@ def plotfits(self, res1, res2=None, save_plot=False,
                 plt.savefig(namestr + '_ps_fit.png', format='png')
 
         return
+
+def calc_likelihood(measured_vals, updated_model, weights, x, m=1):
+    pll = PSDLogLikelihoodBound(x, measured_vals, updated_model, m=m)
```

Comment:
I'm confused by this function. What's its purpose? Why not just use the class definition directly? Also, I don't think `PDLogLikelihoodBound` should be hard-coded here.